src/components/item.jsx

```diff
@@ -24,13 +27,27 @@ class Item extends React.Component {
     this.setState({
       itemId: this.props.itemId
```

Comment:
Maybe I'm misunderstanding, but why do you need to make this `this.setState` call if you're setting this state at initialization on line 11? I'm not sure I know a way to change the prop without rerendering the entire component.

Reply:
I can probably change these back to a default value. I'll test it.